app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt

```diff
@@ -719,7 +761,29 @@ class PinPasswordActivityTest {
   }
 
   @Test
-  fun testPinPassword_withAdmin_showHidePassword_imageChangesToHide() {
+  fun testPinPassword_withAdmin_clickShowHideIcon_hasPasswordShownContentDescription() {
+    ActivityScenario.launch<PinPasswordActivity>(
+      PinPasswordActivity.createPinPasswordActivityIntent(
+        context = context,
+        adminPin = adminPin,
+        profileId = adminId
+      )
+    ).use {
+      closeSoftKeyboard()
+      onView(withId(R.id.show_pin)).perform(click())
```

Comment:
```suggestion
      testCoroutineDispatchers.runCurrent()
      onView(withId(R.id.show_pin)).perform(click())
```
test is failing because less than 90 percent of the view is visible.

Reply:
Tests are passing now. Thanks.